consul/agent.go

```diff
@@ -2,12 +2,18 @@ package consul
 
 import (
 	"fmt"
+	"io/ioutil"
+	"log"
 	"net/http"
+	"os"
 	"time"
 
 	"github.com/hashicorp/consul/api"
 )
 
+var consulACLTokenEnv = "CONSUL_ACL_TOKEN"
```

Comment:
I think it will be more convenient to use default consul library logic for this:
https://github.com/hashicorp/consul/blob/master/api/api.go#L293-L295

Reply:
It's great, I didnt consider that, thanks!